<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>change_pass_form</class>
 <widget class="QWidget" name="change_pass_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>9</y>
     <width>471</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="change_pass_label">
      <property name="text">
       <string>Введите новый пароль:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="change_pass_lineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="change_pass_2_label">
      <property name="text">
       <string>Введите новый пароль еще раз:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="change_pass_2_lineEdit"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="confirm_pushbutton">
        <property name="text">
         <string>Подтвердить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel_pushbutton">
        <property name="text">
         <string>Отмена</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
